angular-app/src/app/pages/admin-panel/admin-panel/admin-panel.component.html

```diff
@@ -1,7 +1,4 @@
-<head>
-  <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
-</head>
-
+<!-- Author:Ariel Camargo-->
```

Comment:
This is introducing an old version of Admin Panel, you need to pull changes from main first before merging this PR